.github/workflows/release.yml

```diff
@@ -18,26 +18,42 @@
 #
 
 name: Release skupper-router
-#on:
-#  release:
-#    types: [published]
 on:
-  workflow_dispatch:
+  push:
+    # Sequence of patterns matched against tags
+    # Right now, we run this job automatically when a semantically versioned
+    # tag is pushed.
+    tags:
+      - '[0-9]+.[0-9]+.[0-9]+'
```

Comment:
Maybe escape the period in the regular expression? Afaik what's there currently would match non-semantic version strings such as `99999` or `0x0y0`.

Reply:
I believe it is not a real regular expression. There are a few characters that must be escaped, but `.` in this case 
is not one of them, see:
https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet